years.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if (grep -r "Copyright \+(c) \+2009-.*rultor.com" \
+  --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . \
+  | grep -v 2009-`date +%Y`) then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+fi
+
+if (grep -r -L "Copyright \+(c) \+2009-.*rultor.com" \
+ --include=*.{java,xml,vm,groovy,txt,fml,properties} --exclude-dir "target" . \
+ | grep -v 2009-`date +%Y`) then
```

Comment:
@mkordas yes. If to remove it we will have next output `Viktors-MBP:_rultor viktor$ ./years.sh 
Files above must have copyright block in header`. And respectively script finishes with error code 1. Agree, it is strange behavior, this was quickest fix.